custom-recipes/api-connect/recipe.json

```diff
@@ -193,7 +198,7 @@
             "description": "",
             "type": "STRING",
             "defaultValue": null,
-            "visibilityCondition": "(['page', 'offset'].indexOf(model.pagination_type)>-1)"
+            "visibilityCondition": "(['offset'].indexOf(model.pagination_type)>-1)"
         },
         {
             "name": "skip_key",
```

Comment:
This parameter seems te same than the next one. Can't it be gathered in a unique parameter? With the following visibility condition:
`"visibilityCondition": "(['page', 'offset'].indexOf(model.pagination_type)>-1)"`